projects/packages/jetpack-mu-wpcom/src/features/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-launchpad.php

```diff
@@ -139,13 +139,11 @@ public function can_access() {
 	public function get_data( $request ) {
 		$checklist_slug = isset( $request['checklist_slug'] ) ? $request['checklist_slug'] : get_option( 'site_intent' );
 
-		return array(
-			'site_intent'        => get_option( 'site_intent' ),
-			'launchpad_screen'   => get_option( 'launchpad_screen' ),
-			'checklist_statuses' => get_option( 'launchpad_checklist_tasks_statuses', array() ),
-			'checklist'          => wpcom_get_launchpad_checklist_by_checklist_slug( $checklist_slug ),
-			'is_enabled'         => wpcom_get_launchpad_task_list_is_enabled( $checklist_slug ),
-			'is_dismissed'       => wpcom_launchpad_is_task_list_dismissed( $checklist_slug ),
+		return wpcom_launchpad_get_checklist_for_api_response(
+			$checklist_slug,
+			get_option( 'site_intent' ),
+			get_option( 'launchpad_screen' ),
+			get_option( 'launchpad_checklist_tasks_statuses', array() )
```

Comment:
What do you think about this?
```suggestion
		return array_merge(
			wpcom_launchpad_get_checklist_for_api_response( $checklist_slug ),
			[
				'site_intent'        => get_option( 'site_intent' ),
				'launchpad_screen'   => get_option( 'launchpad_screen' ),
				'checklist_statuses' => get_option( 'launchpad_checklist_tasks_statuses', array() ),
			]
		);
```